plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/use-form-validation.ts

```diff
@@ -0,0 +1,269 @@
+/**
+ * External dependencies
+ */
+import {
+	KeyedFormField,
+	FormFields,
+	AdditionalValues,
+	BillingAddress,
+	ShippingAddress,
+} from '@woocommerce/settings';
+import {
+	useSchemaParser,
+	DocumentObject,
+	usePrevious,
+} from '@woocommerce/base-hooks';
+import type { JSONSchemaType, ErrorObject } from 'ajv';
+import { __, sprintf, getLocaleData } from '@wordpress/i18n';
+import { useRef } from '@wordpress/element';
+import fastDeepEqual from 'fast-deep-equal/es6';
+import { isPostcode } from '@woocommerce/blocks-checkout';
+import { isEmail } from '@wordpress/url';
+
+const getFieldLabelCasing = ( fieldLabel: string ): string => {
```

Comment:
Can we add a doc block why this is needed? It seems like it might be a pain to maintain if its needed.